### DISCUSSION TITLE
In master detail relationship I need to print Excel Report in master grid containing data from both master and detail object

### DISCUSSION THREAD

**Discussion by ziaatrix** [serenity-is/Serenity#7223] [General]
Hi @VictorTomaili,

In master detail relationship I need to print Excel Report in master grid containing data from both master and detail object

For example
Master is Order
Detail is Order Line Item

In Order Line Item there are fields such as Product, Price, Quantity, Total
In Order there are fields such as Date, Customer

In Order Grid I wanted to generate in Excel the combined above fields both from master and detail. Any idea please?

**Comment by ziaatrix** [comment]
@VictorTomaili and @volkanceylan any idea please? Thank you.

**Comment by VictorTomaili** [comment]
Don't mention us for a reply. Use reports for that.

**Comment by ziaatrix** [comment]
Hi @VictorTomaili 

Could you please provide which report are you referring to. As of now I can see only excel report which is printing based on the grid data

**Reply by VictorTomaili** [reply]
We didn't have such as example. You should write own report with your data based on existing samples.

**Reply by ziaatrix** [reply]
@VictorTomaili ,

Thank you for your reply. I've seen this post, same thing what I need however the code might be already outdated as per the current structure of serenity. Could you please check and give us your suggestion. Thank you so much.
